library/classes/Installer.class.php

```diff
@@ -472,6 +472,8 @@ public function create_site_directory()
                 $this->error_message = "unable to copy directory: '$source_directory' to '$destination_directory'. " . $this->error_message;
                 return false;
             }
+            // the new site will create it's own keys so okay to delete these copied from the source site
+            array_map('unlink', glob($destination_directory . "/documents/logs_and_misc/methods/*"));
```

Comment:
Two scenarios.
If cloning the database, then want to keep the keys.
If not cloning the database, then should remove the keys.

Reply:
![image](https://user-images.githubusercontent.com/278968/161184691-8ff502a1-bb33-4de2-a8c6-c8e586f77ed8.png)

![image](https://user-images.githubusercontent.com/278968/161184731-e8fa2215-dc80-4025-8ac3-65c3ce7929b2.png)